sky/engine/core/painting/Rect.dart

```diff
@@ -19,6 +19,9 @@ class Rect {
   }
 
   /// Construct a rectangle from its left and top edges, its width, and its height.
+  ///
+  /// To construct a [Rect] from a [Point] or [Offset] and a [Size], you can use
+  /// the rectangle constructor operator `&`. See [Point.&] and [Offset.&].
```

Comment:
There's no point&point operator, it's point&size. That's why I put it here and not the other one. I don't want people to try to shoe-horn their code into using & when they have two point...